addons/website/static/src/snippets/s_google_map/google_map.js

```diff
@@ -0,0 +1,104 @@
+/* GLOBAL VARIABLE : google */
+
+import { Interaction } from "@website/core/interaction";
+import { registry } from "@web/core/registry";
+
+export class GoogleMap extends Interaction {
+    static selector = ".s_google_map";
+    dynamicContent = {
+        _window: {
+            "t-on-resize": () => { if (this.gps) { map.setCenter(this.gps) } },
```

Comment:
map is undefined here. i assume you meant `this.map`, but then you forgot to set it

Reply:
also, be careful, if `this.map` is not set (for example, because google object is not defined, then this code could crash whenever the user is resizing the window. need to defend against that